modules/install-roxctl-cli-linux.adoc

```diff
@@ -9,11 +9,12 @@ You can install the `roxctl` CLI binary on Linux by using the following procedur
 
 .Procedure
 
-. Download the latest version of the `roxctl` CLI:
+. Download the `roxctl` CLI (supported architectures are `amd64`, `ppc64le` and `s390x`):
```

Comment:
```suggestion
. Download the `roxctl` CLI, which supports the `amd64`, `ppc64le`, and `s390x` architectures:
```

Reply:
I wanted to clarify about this one. Technically it is not one executable that supports three architectures. Rather, these are three different executables, one per architecture. Would it be still fair to say that "roxctl supports amd64, ppcl64le, and s390x architectures"?

Reply:
yeah, makes sense, perhaps we can add it as a NOTE just after the command. something like:

> . Download the `roxctl` CLI:
> <command>
> NOTE: `roxctl` CLI supports `amd64`, `ppcl64le`, and `s390x` architectures.

You can update this at all the instances.

Reply:
Moved to the `NOTE`. Yet, I keep changing the message. What do you think about the current one?